index.html

```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="sidebar">
+          <li onclick=hideSidebar()><button class="nav-button"><i class="fa-solid fa-x" style="color: white; font-size: 1em;"></i></button></li>
```

Comment:
Add aria-label for screen readers:

```suggestion
          <li onclick=hideSidebar()><button class="nav-button" aria-label="Close sidebar"><i class="fa-solid fa-x" style="color: white; font-size: 1em;"></i></button></li>
```

Reply:
Also, move the inline style into your CSS file